test/evergreen/find_cmake.sh

```diff
@@ -40,6 +40,7 @@ find_cmake ()
         # Some images have no cmake yet, or a broken cmake (see: BUILD-8570)
         echo "-- MAKE CMAKE --"
         CMAKE_INSTALL_DIR=$(readlink -f cmake-install)
+        if [ -d  cmake-3.11.0 ]; then rm -r cmake-3.11.0; fi
```

Comment:
Can we make the CMake version (3.11) a variable in this script? It will make this script easier to update if required in the future.

Reply:
Done: f70bf23acb506305ccbfb9aff3779df788c0f0aa